incubator/hnc/api/v1alpha2/hierarchy_types.go

```diff
@@ -44,25 +42,32 @@ const (
 	AnnotationNoneSelector = AnnotationPropagatePrefix + "/none"
 )
 
-// Condition codes. *All* codes must also be documented in the comment to Condition.Code, be
-// inserted into AllCodes, and must have an entry in ClearConditionCriteria, set in init() in this
-// file.
-//
-// Please keep this list in alphabetic order.
 const (
-	CritAncestor              Code = "CritAncestor"
-	CritCycle                 Code = "CritCycle"
-	CritDeletingCRD           Code = "CritDeletingCRD"
-	CritParentMissing         Code = "CritParentMissing"
-	SubnamespaceAnchorMissing Code = "SubnamespaceAnchorMissing"
+	// Condition types.
+	ConditionActivitiesHalted string = "ActivitiesHalted"
+	ConditionBadConfiguration string = "BadConfiguration"
+
+	// Condition reasons.
+	HaltedReasonAncestor            string = "AncestorHaltActivities"
```

Comment:
Nit (if you're making other changes anyway): I'd take out the "halted" prefix, I think "reason" is fine.